pkgs/development/python-modules/django-formtools/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, django }:
+
+buildPythonPackage rec {
+  pname = "django-formtools";
+  version = "2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1chkbl188yj6hvhh1wgjpfgql553k6hrfwxzb8vv4lfdq41jq9y5";
+  };
+
+  doCheck = false; # require tox and multiple python versions
```

Comment:
Missing pythonImportsCheck.

Reply:
Same here, pypi package does not embed test code